sssd_test_framework/markers.py

```diff
@@ -56,18 +61,39 @@ def pytest_runtest_setup(item: pytest.Item) -> None:
     if not isinstance(item, pytest.Function):
         raise TypeError(f"Unexpected item type: {type(item)}")
 
+    topology: list[Topology] = []
+    mh: MultihostItemData | None = MultihostItemData.GetData(item)
```

Comment:
mh_item_data so one does not confuse it with mh which is the MultihostFixture

Reply:
Fixed